src/Caching/Hybrid/src/Runtime/HybridCacheEntryFlags.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Extensions.Caching.Hybrid;
+
+/// <summary>
+/// Additional flags that apply to a <see cref="HybridCache"/> operation
+/// </summary>
+[Flags]
+public enum HybridCacheEntryFlags
+{
+    /// <summary>
+    /// No additional flags
+    /// </summary>
+    None = 0,
+    /// <summary>
+    /// Do not read from the local in-process cache
+    /// </summary>
+    DisableLocalCacheRead = 1 << 0,
+    /// <summary>
+    /// Do not write to the local in-process cache
+    /// </summary>
+    DisableLocalCacheWrite = 1 << 1,
+    /// <summary>
+    /// Do not use the local in-process cache for reads or writes
+    /// </summary>
+    DisableLocalCache = DisableLocalCacheRead | DisableLocalCacheWrite,
+    /// <summary>
+    /// Do not read from the secondary distributed cache
+    /// </summary>
+    DisableDistributedCacheRead = 1 << 2,
+    /// <summary>
+    /// Do not write to the secondary distributed cache
+    /// </summary>
+    DisableDistributedCacheWrite = 1 << 3,
+    /// <summary>
+    /// Do not use the local in-process cache for reads or writes
+    /// </summary>
+    DisableDistributedCache = DisableDistributedCacheRead | DisableDistributedCacheWrite,
+    /// <summary>
+    /// Only fetch the value from cache - do not attempt to access the underlying data store
+    /// </summary>
+    DisableUndelyingData = 1 << 4,
```

Comment:
How come this one isn't split into read and write?

Reply:
@amcasey  this is the final "I have nothing, go to the whatever data service and get it" - we can't write to that

Reply:
Part of me wants to stick a "read" in there for parallelism, but I'll defer to your judgement.